build/generic/web/viewer.html

```diff
@@ -240,6 +240,7 @@
           <menuitem id="contextPageRotateCw" label="Rotate Clockwise"
```

Comment:
I don't think so, since we're still allowing users to use the equivalent button.

Now that I think about it, I'm not sure if `class="hidden"` works on `<menuitem>`s anyway.